### ISSUE TITLE
NTR: beta-glucuronidase high in blood (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8235]
New term request beta-glucuronidase high in blood (elthat_190712173759):   UMLS: 5195139

**Comment by pnrobinson** [comment]
@MickeySegal
is this enzyme typically measured in blood/serum?
In MPS it is measured in fibroblasts or leukocytes
https://omim.org/entry/253220

**Comment by MickeySegal** [comment]
https://www.omim.org/clinicalSynopsis/618392
"Elevated beta-glucuronidase"

and in the main article for the disease
https://www.omim.org/entry/618392
"Elevated levels of blood lysosomal enzymes have also been observed"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000819
- label: Elevated circulating beta-glucuronidase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:30046013
- def: The activity of beta-glucuronidase (EC 3.2.1.31) in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8235
- synonyms:
